tests/php_unit_converter/test.sh

```diff
@@ -0,0 +1,199 @@
+        fi
+   done
+}
+
+check_file_content2() {    
+    for currentFile in $path_to_php_folder/*
+    do
+        if grep  "yard"  $currentFile && grep "1.094" $currentFile
+        then
+            content_exists=1
+            break     
+        else
+            content_exists=0
+            echo -e "${BOLD}Checking the sql file ...${NONE}"
+            echo -e "1. Ensure your sql file contains the accident_cases table and has a primary key"${NONE}
+        fi
+   done
+}
+
+check_file_content3() {    
+    for currentFile in $path_to_php_folder/*
+    do
+        if grep  "pound"  $currentFile && grep "2.205" $currentFile
+        then
+            content_exists=1
+            break     
+        else
+            content_exists=0
+            echo -e "${BOLD}Checking the sql file ...${NONE}"
+            echo -e "1. Ensure the accident_cases and passengers_details tables has a foreign key."${NONE}
+        fi
+   done
+}
+
+check_file_content4() {    
+    for currentFile in $path_to_php_folder/*
+    do
+        if grep "fahrenheit"  $currentFile && grep "1.8" $currentFile
+        then
+            content_exists=1
+            break     
+        else
+            content_exists=0
+            echo -e "${BOLD}Checking the sql file ...${NONE}"
+            echo -e "1. Ensure the passengers table contains the age column"${NONE}
+        fi
+   done
+}
+
+check_file_content5() {    
+    for currentFile in $path_to_php_folder/*
+    do
+        if grep "32"  $currentFile && grep "=" $currentFile
+        then
+            content_exists=1
+            break     
+        else
+            content_exists=0
+            echo -e "${BOLD}Checking the sql file ...${NONE}"
+            echo -e "1. Ensure you create a database"${NONE}
+        fi
+   done
+}
+
+write_json_response() {
+    echo "{" > $path_to_log
+    echo "  \"stats\": {" >> $path_to_log
+    echo "    \"tests\": $total_tests," >> $path_to_log
+    echo "    \"passes\": $no_of_passes," >> $path_to_log
+    echo "    \"failures\": $no_of_failures," >> $path_to_log
+    echo "    \"owner\": \"$folder_name\"" >> $path_to_log
+    echo "  }" >> $path_to_log
+    echo "}" >> $path_to_log
+}
+
+write_output() {
+    echo ""
+    echo -e "${BOLD}Test Results${NONE}"
+    echo -e "${ITALIC}Total number of tests:${NONE} ${BOLD}$total_tests${NONE}"
+    echo -e "${GREEN}${ITALIC}Passed tests:${NONE} ${BOLD}$no_of_passes${NONE}"
+    echo -e "${RED}${ITALIC}Failed tests:${NONE} ${BOLD}$no_of_failures${NONE}"
+}
+
+no_of_passes=0
+no_of_failures=0
+
+    check_database_folder_existence
+
+    if [ $database_folder_exists -eq 1 ]
+    then
+        no_of_passes=$((no_of_passes+1))
+
+        # check_file_existence
+
+        # if [ $file_exists -eq 1 ]
+        # then
+        #     no_of_passes=$((no_of_passes+1))
+     
+            check_file_content1
+
+            if [ $content_exists -eq 1 ]
+            then
```

Comment:
I'm not sure about the logic here. Could you help format it better for readability? Thanks